jbpm-services/jbpm-kie-services/src/test/java/org/jbpm/kie/services/test/RuntimeDataServiceImplTest.java

```diff
@@ -1038,7 +1038,6 @@ public void testGetTaskByIdWithCorrelation() {
         assertNotNull(userTask);
         assertCorrelationAndProcess(userTask, correlationKey);
         assertEquals(processInstanceId, userTask.getProcessInstanceId());
-        processService.abortProcessInstance(processInstanceId);
```

Comment:
not sure why do we need to remove this line? Don't we have to abort instances before removing data from the tables?
Also, same method will be called in the `@after` method cleanup once this test is finished.

Reply:
In the after method, it will be aborted as other tests, so it's not strictly needed. Before this fixing, it was necessary for CDI tests to avoid failing and it was the main clue for understanding what was happening.